Monika After Story/game/script-apologies.rpy

```diff
@@ -0,0 +1,250 @@
+        6: "missing Christmas.",
+        7: "forgetting your birthday.",
+        8: "not spending time with you on your birthday.",
+        9: "the game crashing.",
+        10: "the game crashing." #easiest way to handle this w/o overrides
+    }
+
+    #Set the prompt for this...
+    if persistent._mas_apology_time_db != {}:
+        #If there's a non-generic apology reason pending we use "for something else."
+        $ mas_getEV('mas_apology_generic').prompt = "...for " + player_apology_reasons.get(mas_apology_reason,player_apology_reasons[0])
+    else:
+        #Otherwise we just use "for something."
+        $ mas_getEV('mas_apology_generic').prompt = "...for " + player_apology_reasons.get(mas_apology_reason,"something.")
+
+    #Then we delete this since we're not going to need it again until we come back here, where it's created again.
+    #No need to store excess memory
+    $ del player_apology_reasons
+
+    #Now we run through our apology db and find what's unlocked
+    python:
+        apologylist = [
+            (mas_getEV(ev).prompt,ev,False,False)
+            for ev in store.mas_apology.apology_db
+            if mas_getEV(ev).unlocked
+        ]
+
+        #The back button
+        return_prompt_back = ("Nevermind", False, False, False, 20)
+
+    #Display our scrollable
+    show monika at t21
+    call screen mas_gen_scrollable_menu(apologylist,(evhand.UNSE_X, evhand.UNSE_Y, evhand.UNSE_W, 500), evhand.UNSE_XALIGN, return_prompt_back)
+    show monika at t11
+
+    #Make sure we don't lose this value
+    $ apology =_return
+
+    #Handle backing out
+    if not apology:
+        if mas_apology_reason is not None or persistent._mas_apology_time_db != {}:
+            if mas_isMoniAff(higher=True):
+                m 1ekd "[player], if you're feeling guilty about what happened..."
+                m 1eka "You don't have to be afraid of apologizing, we all make mistakes after all."
+                m 3eka "We just have to accept what happened, learn from our mistakes, and move on, together. Okay?"
+            elif mas_isMoniNormal(higher=True):
+                m 1eka "[player]..."
+                m "If you want to apologize, go ahead. It'd mean a lot to me if you did."
+            elif mas_isMoniUpset():
+                m 2rkc "Oh..."
+                m "I was kind of--"
+                $ _history_list.pop()
+                m 2dkc "Nevermind."
+            elif mas_isMoniDis():
+                m 6rkc "...?"
+            else:
+                m 6ckc "..."
+        else:
+            if mas_isMoniAff(higher=True):
+                m 1hua "Ehehe, you don't have anything to be sorry about silly~"
+            elif mas_isMoniNormal(higher=True):
+                m 1eka "I'm not sure what you wanted to apologize for, but I'm sure that it was nothing."
+
+        return
+
+
+    #Call our apology label
+    #NOTE: mas_setApologyReason() ensures that this label exists
+    $ renpy.call(apology)
```

Comment:
```suggestion
    call expression apology
```
renpy.call works somewhat differently than the call statement. dont use it unless its absolutely the only way to do something.